src/components/WorkspaceStatusLabel/WorkspaceStatusLabel.tsx

```diff
@@ -11,18 +11,31 @@
  */
 
 import {
-  BanIcon,
-  ErrorCircleOIcon,
+  ExclamationCircleIcon,
```

Comment:
Please rename this component as `WorkspaceStatusLabel/index.tsx`.

Reply:
I have fixed it.